protocol-units/sequencing/memseq/sequencer/src/lib.rs

```diff
@@ -99,13 +99,16 @@ impl<T: MempoolTransactionOperations> Sequencer for Memseq<T> {
 
 	/// Waits for the next block to be built, either when the block size is reached or the building time expires.
 	async fn wait_for_next_block(&self) -> Result<Option<Block>, anyhow::Error> {
+		info!(target: "movement_timing",  "CALLED wait_for_next_block");
 		let mut transactions = Vec::with_capacity(self.block_size as usize);
 
 		let now = Instant::now();
 
 		loop {
 			let current_block_size = transactions.len() as u32;
 			if current_block_size >= self.block_size {
+				info!("condition `current_block_size >= self.block_size ` truthy");
```

Comment:
This could actually describe the semantics:
```suggestion
				debug!("block is above the size limit");
```